diff --git a/dist/module.mjs b/dist/module.mjs
index bd2dd6975bb57752abe52060b9a447dae586bdeb..aea7e1f386d02b848b520f156649faa7995b7247 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -400,8 +400,10 @@ const module = defineNuxtModule({
     addImportsDir(ctx._resolver.resolve("./runtime/composables"));
     for (const part of parts)
       await part(ctx);
-    nitro.publicAssets = nitro.publicAssets || [];
-    nitro.publicAssets.push({ dir: ctx._buildDir, baseURL: "/" });
+    if (nitro) {
+      nitro.publicAssets = nitro.publicAssets || [];
+      nitro.publicAssets.push({ dir: ctx._buildDir, baseURL: "/" });
+    }
   }
 });
 